<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACG_LeiClass</class>
 <widget class="QMainWindow" name="ACG_LeiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ACG_Lei</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuSubdivision">
    <property name="title">
     <string>Subdivision</string>
    </property>
    <addaction name="actionLoop"/>
    <addaction name="actionButterfly"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSubdivision"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoop">
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="actionButterfly">
   <property name="text">
    <string>Butterfly</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ACG_Lei.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>ACG_LeiClass</receiver>
   <slot>action_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>ACG_LeiClass</receiver>
   <slot>action_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoop</sender>
   <signal>triggered()</signal>
   <receiver>ACG_LeiClass</receiver>
   <slot>action_loop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButterfly</sender>
   <signal>triggered()</signal>
   <receiver>ACG_LeiClass</receiver>
   <slot>action_butterfly()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action_open()</slot>
  <slot>action_save()</slot>
  <slot>action_loop()</slot>
  <slot>action_butterfly()</slot>
 </slots>
</ui>
